### DISCUSSION TITLE
Project Name Ideas

### DISCUSSION THREAD

**Discussion by JWoodbury125** [FS-Capstone/Capstone#94] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JWoodbury125** [comment]
Project Name Suggestions:

**Comment by JanaeHijaz** [comment]
FullStock (scott's idea lol)

**Comment by GingerVitas** [comment]
Waste Not (Tagline: Fighting food waste one kitchen/pantry/home/whatever at a time)
What's for Dinner?
